js/app/test/blocks_chained_events.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { test, expect } from "@gradio/tootils";
+
+test("Success event", async ({ page }) => {
+	const textbox = page.getByLabel("Result");
+	await expect(textbox).toHaveValue("");
+
+	await Promise.all([
+		page.click("text=Trigger Failure"),
+		page.waitForResponse("**/run/predict")
+	]);
+	// Since the event is not triggered, the value should not change
```

Comment:
See my comment below about comments in tests.

Reply:
Done!